endpoints-support.md

```diff
@@ -18,8 +18,8 @@
 
 ## HPE OneView
 
-| Endpoints                                                                       | Verb     | V200 | V300 | V500 | V600 | V800
-| --------------------------------------------------------------------------------------- | -------- | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: |
+| Endpoints                                                                       | Verb     | V200 | V300 | V500 | V600 | V800 | V1000 | V1200 
```

Comment:
Let's delete the older API versions(200,300,500 and 600) from the endpoints support file to make visible the newer API versions.